### ISSUE TITLE
this is a bug report

### ISSUE THREAD

**Issue by danielroe** [nuxt/nuxt#26910]
super bad bug but I am tired and don't feel like providing a reproduction.

pls fix asap

**Comment by daniluk4000** [comment]
Yeah it's only reproduced in my project and nowhere else but I can't share because of NDA. We've lost millions and it was only yesterday because no fix
